database/src/main/java/com/firebase/ui/database/FirebaseListAdapter.java

```diff
@@ -27,21 +29,21 @@
 public abstract class FirebaseListAdapter<T> extends BaseAdapter implements FirebaseAdapter<T> {
     private static final String TAG = "FirebaseListAdapter";
 
-    protected final Activity mActivity;
+    protected final Context mContext;
```

Comment:
@samtstern yeah, once I saw the conflicts I realized this PR was built using personal commits instead of dev. Hopefully it won't be too tough to resolve them! 😀